app/src/app/[locale]/serverAction/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { useFormState } from "react-dom";
```

Comment:
Just a heads up: I think this is now [`useActionState`,](https://react.dev/reference/react/useActionState) although I don't know if it's available yet in the version of React that's shipped with Next.js 14. If it's not, we might want to have a backlog ticket to update this when it's available.

Reply:
Thanks - I believe we would need to update to Next 15 RC  (React 19) to get that new hook? Maybe we should wait until its out of the final RC stage and into stable?

Could probably upgrade to next 14.2 on this project though.